src/bin/tools/downloadAndUnzip.ts

```diff
@@ -71,14 +69,24 @@ async function getNpmProxyConfig(): Promise<Pick<FetchOptions, "proxy" | "noProx
  * @returns promise for the full path of the downloaded file
  */
 async function download(url: string, dir: string, filename: string): Promise<string> {
-    const proxyOpts = await getNpmProxyConfig();
-    const cacheRoot = process.env.XDG_CACHE_HOME ?? homedir();
-    const cachePath = join(cacheRoot, ".keycloakify/cache");
-    const opts: FetchOptions = { cachePath, ...proxyOpts };
-    const response = await fetch(url, opts);
+    console.log(`Downloading ${url} to ${dir}...`);
+
     const filepath = pathJoin(dir, filename);
     await mkdir(dir, { recursive: true });
+
+    if (await maybeStat(filepath)) {
+        console.log(`File ${filepath} already exists, skipping download.`);
+        return filepath;
+    }
+
+    const proxyOpts = await getNpmProxyConfig();
+
+    const response = await fetch(url, proxyOpts);
+
     await writeFile(filepath, response.body);
+
+    console.log(`Downloaded ${url} to ${filepath}!`);
```

Comment:
should this stay in production code? everywhere else @garronej used custom logger

Reply:
@garronej would you consider introducing https://www.npmjs.com/package/debug? I've been using it for a long time and its super handy cause you don't need to pass a config from module to module